diagnostics-app/src/app/app-routing.module.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright 2021 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+
```

Comment:
In each of these files, please follow Google's style guide: https://google.github.io/styleguide/jsguide.html#source-file-structure
For example, each file should start with @fileoverview {description}, e.g:
```
/**
  * @fileoverview Description of file, its uses and information
  * about its dependencies.
  * Note: Auto-generated file.
  */
```

Reply:
Done these changes in PR #4